src/main/java/mate/academy/springboot/datajpa/service/impl/CategoryServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.springboot.datajpa.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.repository.CategoryRepository;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class CategoryServiceImpl implements CategoryService {
+    private CategoryRepository categoryRepository;
```

Comment:
Somewhere in your program dependencies are `final` but somewhere are not. Follow the same type of code style everywhere in your program